style.css

```diff
@@ -1 +1,182 @@
-/*  Add your styling here */
+.hamIcon{
+    position: absolute;
+    right: 10px;
+    top: 20px;
+}
+
+.logo_image{
+    display: flex;
+    justify-content: center;
+    background: url("images/cake-logo.jpg");
+    height: 200px;
+    background-position: center;
+}
+.nav_bar{
+    background: url("images/nav_back.JPG");
+    background-repeat: repeat-x;
+    background-size: cover;
+}
+
+.mainMenu{
+    font-family: 'Courgette', cursive;
+    right:2px;
+    top: 100px;
+    padding: 30px;
+    font-size: 30px;
+    color: black;
+    height: 200px;
+    border-radius:10px;
+    border:4px double plum;
+    text-decoration: none;
+    background: url("images/bar-background.jpg");
+
+}
+
+body{
+    background: url("images/wallpaper.jpg");
+}
+
+a:hover{
+    color:blue;
+    text-decoration: none;
+}
+.My_title{
+    color:#fc0384;
+    font-size: 50px;
+    font-family: 'Courgette', cursive;
+    /*display: flex;
+    flex-direction: column;
+    justify-content:center;*/
+    text-align: center;
+    padding: 50px 20px 10px 50px;
+
+}
+.description{
+    font-size: 30px;
+    text-align: center;
+    color: black;
+    font-family: 'Courgette', cursive;
+    padding: 0 auto;
+    margin: 10 10;
+}
+.main_cakes{
+    display:flex;
+    flex-direction:column;
+    border:2px solid pink;
+    border-radius: 10px;
+    border-style: ridge;
+    box-sizing:border-box;
+}
+
+.big_pic{
+    display: flex;
+    flex-direction: column;
+    justify-content:center;
+    background-position: center;
+    background-repeat: no-repeat;
```

Comment:
This won't do anything if you don't provide a background-image. You've chosen to use an `img` tag inside, which is fine, but you need to add a CSS rule to that `img` so that it is only 100% of its parent.

Reply:
thank you lucy,I dont understand which rule i should use to make my image as parent